### PR TITLE
first check

### PR DESCRIPTION
**Pull request by MichaelMorami**

### CODE REVIEW

File: reports/Certora_Extra_Checks.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## proposal 3:
+- For each market:
+    - checkd that the correct market was called (Ethereum V2 or V3, etc, depending on the payload)
+    - We checked that the values that were declared to be updated in the proposal were indeed adjusted to on the correct asset and to the declared values, i.e. slope 1 was changed to 6% (6_00) 
+    - We checked that no other parameters were changed on this proposal, i.e. all other values were kept the same
+
+
+## Questions:
```
Comment (eboadom) [inline reports/Certora_Extra_Checks.md:28]: The only way is asking the proposer

File: reports/Certora_Extra_Checks.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- On avalanche (and other chains) how can we know that the assets being updated are the ones intended? the ipfs states for example usdc but the implementation refers to usdc.e. How can we know that is what intended?
+- Seatbelt of metis is broken link, what should be done?
```
Comment (eboadom) [inline reports/Certora_Extra_Checks.md:30]: Metis is not covered by seatbelt, so the analysis should be more manual, checking the payload and verifying its correctness with the create2 strategy

File: reports/3-aave-v2-v3-multichain-curve-IR-curves-update.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+* Base V3 - [https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/payloads/8453/0x2DC219E716793fb4b21548C0f009Ba3Af753ab01/6.md](https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/payloads/8453/0x2DC219E716793fb4b21548C0f009Ba3Af753ab01/6.md)
+
+* Metis V3 - []()
+
+* Gnosis V3 - [https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/payloads/100/0x9A1F491B86D09fC1484b5fab10041B189B60756b/3.md](https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/payloads/100/0x9A1F491B86D09fC1484b5fab10041B189B60756b/3.md)
+
+<br>
+
+### Technical analysis
```
Comment (eboadom) [inline reports/3-aave-v2-v3-multichain-curve-IR-curves-update.md:81]: Here better to have all changes summarized

File: reports/2-aave-v2-v3-avalanche-reconfigurating-wbtce.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+[https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/proposals/2.md](https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/proposals/2.md)
+
+**Payload reports**
+
+[https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/payloads/43114/0x1140CB7CAfAcC745771C2Ea31e7B5C653c5d0B80/11.md](https://github.com/bgd-labs/seatbelt-gov-v3/blob/main/reports/payloads/43114/0x1140CB7CAfAcC745771C2Ea31e7B5C653c5d0B80/11.md)
+
+
+<br>
+
+### Technical analysis
```
Comment (eboadom) [inline reports/2-aave-v2-v3-avalanche-reconfigurating-wbtce.md:68]: Here the changes should be summarized

File: reports/Certora_Extra_Checks.md (hunk 1 above)
Comment (MichaelMorami) [reply reports/Certora_Extra_Checks.md:28]: On V3 the engine has the option of KEEP_SAME_VALUE or something along those lines.
On V2 we checked the values on the block explorer to ensure the liquidity bonus is the same as they set there

File: reports/3-aave-v2-v3-multichain-curve-IR-curves-update.md (hunk 3 above)
Comment (MichaelMorami) [reply reports/3-aave-v2-v3-multichain-curve-IR-curves-update.md:81]: all the the changes from the IPFS? just copy the table?

File: reports/2-aave-v2-v3-avalanche-reconfigurating-wbtce.md (hunk 4 above)
Comment (MichaelMorami) [reply reports/2-aave-v2-v3-avalanche-reconfigurating-wbtce.md:68]: you mean copy the ipfs table?

**Pull request closed without merging**